src/main/java/seedu/address/experimental/model/ReadOnlyEntities.java

```diff
@@ -13,4 +13,6 @@ public interface ReadOnlyEntities {
      * The list will not contain duplicate items.
      */
     ObservableList<Entity> getEntityList();
+
+    Entity getEntityWithName(String name);
```

Comment:
Must this be added to the interface? The search for the Entity can be done within the logic component after getting the entity list from the model.

Reply:
maybe can rename for slight clarity. it does make sense for the model to provide this functionality though.

```suggestion
    Entity getEntityFromName(String name);
```